gen/gen.go

```diff
@@ -46,6 +46,8 @@ func Generate(proto *schema.WebRPCSchema, target string, config *Config) (out *G
 		WebrpcTarget     string
 		WebrpcErrors     []*schema.Error
 		Opts             map[string]interface{}
+		TmplVersion      string
+		TmplTarget       string
```

Comment:
Can we call this
```suggestion
		TargetName      string
		TargetVersion       string
```
to match the `-target=` CLI option?